prep-javascript-dom-creation/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript DOM Creation</title>
+</head>
+<body>
+  <div id="container">
+
+  </div>
+  <script>
+    // Your code here
+    const $container = document.querySelector('#container');
+    console.log($container);
+
+    const $heading = document.createElement('h1');
+    document.getElementsByTagName('$heading').textContent = ('Hello World!');
+    console.log($heading);
```

Comment:
Hi Brennon, try adding the text content to the `$heading` you created on line 15

```js
$heading.textContent = 'Hello World!';
```